docs: add docs community section to onboarding guide (#959)
asyncapi/community: docs/onboarding-guide/docs-community.md

> # Docs community

Reviewer: ```suggestion
# AsyncAPI docs community

```

> ## Open community meetings

Reviewer: ```suggestion
## Open docs community meetings

```

We host regular community meetings via Zoom where we discuss topics relating to the AsyncAPI initiative. All the meetings are livestreamed on our social media channels.

> ### Join meetings

Reviewer: ```suggestion
### Join docs meetings

```

- Add the AsyncAPI calendar to your [Google Calendar](https://calendar.google.com/calendar/u/3?cid=Y19xOXRzZWlnbG9tZHNqNm5qdWh2YnB0czExY0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or download the [ICS file](https://calendar.google.com/calendar/ical/c_q9tseiglomdsj6njuhvbpts11c%40group.calendar.google.com/public/basic.ics) to stay informed about upcoming meetings and calls.

- If you're interested in the `spec` docs, join the dedicated **Spec 3.0 Docs** calls held bi-weekly every Thursday. 

- You can also find [upcoming and past meetings details](https://www.asyncapi.com/community/events) on the events page. 

> ### Watch past meetings

Reviewer: ```suggestion
### Watch past AsyncAPI docs meetings

```

Visit the [AsyncAPI YouTube channel](https://www.youtube.com/AsyncAPI) for past meetings recordings.

> ### Schedule meetings

Reviewer: ```suggestion
### Schedule docs meetings

```

> Read our FAQ for [meeting setup and automation details](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).

Reviewer: ```suggestion
Read our FAQ to [set up and automate your own docs meetings](https://github.com/asyncapi/community/blob/master/MEETINGS_ORGANIZATION.md).

```

> ### Community discussion

Reviewer: ```suggestion
### Docs and education community discussions

```

Visit the [Docs & Education discussion board](https://github.com/orgs/asyncapi/discussions/categories/docs-education) to discuss docs-related issues.

> ## Slack workspace (Asynchronous communication)

Reviewer: ```suggestion
## AsyncAPI Slack workspace and docs channels

```

Visit the [public AsyncAPI Docs & Education discussion board](https://github.com/orgs/asyncapi/discussions/categories/docs-education) to discuss docs-related issues.

## AsyncAPI Slack workspace and docs channels

Join our documentation-specific [Slack](https://www.asyncapi.com/slack-invite) channels to meet other technical writers:

- **#13_docs:** Required for all technical writers. You will find general discussions, questions, ideas, and open `good first issues.`
- **#spec:** If you’re interested in working on `spec` related documentation, connect with Spec subject matter experts to learn more about Spec docs.
- **#tooling:** If you’re interested in working on the [tools related docs](https://www.asyncapi.com/docs/tools), join this channel.

## AsyncAPI documentation roadmap 2023

**Note:** Applications for the 2023 paid writing programs are currently closed. However, we welcome you to join us as a regular docs contributor. Stay tuned for new opportunities on our [Docs’ Community discussions](https://github.com/orgs/asyncapi/discussions/categories/docs-education) or check out the [AsyncAPI Docs Project Board](https://github.com/orgs/asyncapi/projects/12).

> 

Reviewer: Please add links in the table projects of the tools' docs, since right now only Modelina docs have a link to them 😸

Author: Added!
